static/app/views/insights/common/components/widgets/hooks/useResourceLandingSeries.tsx

```diff
@@ -0,0 +1,73 @@
+import type {PageFilters} from 'sentry/types/core';
+import {EMPTY_OPTION_VALUE, MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {getResourceTypeFilter} from 'sentry/views/insights/browser/common/queries/useResourcesQuery';
+import {DEFAULT_RESOURCE_TYPES} from 'sentry/views/insights/browser/resources/settings';
+import {
+  BrowserStarfishFields,
+  useResourceModuleFilters,
+} from 'sentry/views/insights/browser/resources/utils/useResourceFilters';
+import {useSpanMetricsSeries} from 'sentry/views/insights/common/queries/useDiscoverSeries';
+import type {ModuleFilters} from 'sentry/views/insights/common/views/spans/types';
+import {SpanMetricsField} from 'sentry/views/insights/types';
+
+const {SPAN_SELF_TIME, NORMALIZED_DESCRIPTION, SPAN_DOMAIN} = SpanMetricsField;
+const {SPAN_DOMAIN: STARFISH_SPAN_DOMAIN, USER_GEO_SUBREGION} = BrowserStarfishFields;
```

Comment:
had to alias `BrowserStarfishFields.SPAN_DOMAIN` since we moved these down into the new hook and now there are 2 `SPAN_DOMAIN`s in scope.

Reply:
`BrowserStarfishFields` is in the process of being deprecated, could we just grab all of these fields from `SpanMetricsFeild` ?

Reply:
@DominikB2014 Oops, should have looked into the enums... updated!